src/MassiveActionHandler.test.ts

```diff
@@ -110,4 +122,19 @@ describe("MassiveActionHandler", async () => {
       todo_id: 2,
     })
   })
+
+  it("returns a needToSeek block number if state already exists", async () => {
```

Comment:
Can we get some line breaks in here.   Maybe between awaits.  It's very hard to read what's going on in here.